hbase-shell/src/main/ruby/shell/commands/balancer.rb

```diff
@@ -44,7 +44,9 @@ def command(force = nil)
         elsif !force.nil?
           raise ArgumentError, "Invalid argument #{force}."
         end
-        formatter.row([admin.balancer(force_balancer) ? 'true' : 'false'])
+        did_balancer_run = !!admin.balancer(force_balancer)
+        formatter.row([did_balancer_run.to_s])
+        did_balancer_run
```

Comment:
Does it make sense to put this pattern in a method?

Reply:
I think the simple implementation to encapsulate this pattern would be strange. I would rather leave this as is, and revisit the formatter more holistically as a separate ticket.

Reply:
Makes sense, can you create the follow-on issue and leave a short description?

Reply:
Will do! :smile: